notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-092_m1m3_ics_historical_analysis.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<ol><li>I suggest to add a comment to warn that this cell needs 12 minutes to complete the first time</li><li>Would it be possible to add an option to turn off the debug or to at least limit its verbosity ?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>